### PR TITLE
[Merged by Bors] - chore(SetTheory/Ordinal/Arithmetic): `sub_isNormal` → `isNormal_sub`, etc.

### PR DESCRIPTION
**Pull request by vihdzp**
We fix various misnamed lemmas involving `IsNormal` and `IsLimit`.

---

We don't fix `aleph_isNormal` since it will be deprecated anyways in #17674.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/SetTheory/Ordinal/Arithmetic.lean [hunk 1]
```diff
@@ -554,10 +560,13 @@ theorem sub_sub (a b c : Ordinal) : a - b - c = a - (b + c) :=
 theorem add_sub_add_cancel (a b c : Ordinal) : a + b - (a + c) = b - c := by
   rw [← sub_sub, add_sub_cancel]
 
-theorem sub_isLimit {a b} (l : IsLimit a) (h : b < a) : IsLimit (a - b) :=
+theorem isLimit_sub {a b} (l : IsLimit a) (h : b < a) : IsLimit (a - b) :=
```
Comment (YaelDillies) [inline Mathlib/SetTheory/Ordinal/Arithmetic.lean:563]:
```suggestion
theorem IsLimit.sub {a b} (l : IsLimit a) (h : b < a) : IsLimit (a - b) :=
```
no?

File: Mathlib/SetTheory/Ordinal/Arithmetic.lean [hunk 2]
```diff
@@ -479,11 +479,17 @@ theorem add_le_of_limit {a b c : Ordinal} (h : IsLimit b) : a + b ≤ c ↔ ∀
...
-theorem add_isNormal (a : Ordinal) : IsNormal (a + ·) :=
+theorem isNormal_add (a : Ordinal) : IsNormal (a + ·) :=
   ⟨fun b => (add_lt_add_iff_left a).2 (lt_succ b), fun _b l _c => add_le_of_limit l⟩
 
-theorem add_isLimit (a) {b} : IsLimit b → IsLimit (a + b) :=
-  (add_isNormal a).isLimit
+@[deprecated isNormal_add (since := "2024-10-11")]
+alias add_isNormal := isNormal_add
+
+theorem isLimit_add (a) {b} : IsLimit b → IsLimit (a + b) :=
```
Comment (YaelDillies) [inline Mathlib/SetTheory/Ordinal/Arithmetic.lean:488]:
```suggestion
theorem IsLimit.add (a) {b} : IsLimit b → IsLimit (a + b) :=
```

File: Mathlib/SetTheory/Ordinal/Arithmetic.lean [hunk 3]
```diff
@@ -479,11 +479,17 @@ theorem add_le_of_limit {a b c : Ordinal} (h : IsLimit b) : a + b ≤ c ↔ ∀
           · rcases a with ⟨a | a, h₁⟩ <;> rcases b with ⟨b | b, h₂⟩ <;> cases h₁ <;> cases h₂ <;>
               rintro ⟨⟩ <;> constructor <;> assumption⟩
 
-theorem add_isNormal (a : Ordinal) : IsNormal (a + ·) :=
+theorem isNormal_add (a : Ordinal) : IsNormal (a + ·) :=
```
Comment (YaelDillies) [inline Mathlib/SetTheory/Ordinal/Arithmetic.lean:482]:
```suggestion
theorem isNormal_add_right (a : Ordinal) : IsNormal (a + ·) :=
```
looks like a more accurate name

File: Mathlib/SetTheory/Ordinal/Arithmetic.lean [hunk 4]
```diff
@@ -739,23 +748,26 @@ theorem mul_le_of_limit {a b c : Ordinal} (h : IsLimit b) : a * b ≤ c ↔ ∀
         | H β s =>
           exact mul_le_of_limit_aux h H⟩
 
-theorem mul_isNormal {a : Ordinal} (h : 0 < a) : IsNormal (a * ·) :=
+theorem isNormal_mul {a : Ordinal} (h : 0 < a) : IsNormal (a * ·) :=
```
Comment (YaelDillies) [inline Mathlib/SetTheory/Ordinal/Arithmetic.lean:751]: and similarly
```suggestion
theorem isNormal_mul_right {a : Ordinal} (h : 0 < a) : IsNormal (a * ·) :=
```

File: Mathlib/SetTheory/Ordinal/Arithmetic.lean (hunk 1 above)
Comment (vihdzp) [reply Mathlib/SetTheory/Ordinal/Arithmetic.lean:563]: Once I finally get around to replacing `IsLimit` with `IsSuccLimit`, dot notation will break. So I'd rather not introduce it here.

Comment (YaelDillies) [reply Mathlib/SetTheory/Ordinal/Arithmetic.lean:563]: Hmm, okay

Comment (YaelDillies) [reply Mathlib/SetTheory/Ordinal/Arithmetic.lean:563]: Can you still add the `_left` and `_right` as appropriate?

**Comment by YaelDillies** [comment]
maintainer merge

**Comment by b-mehta** [comment]
Thanks!

bors merge

**Pull request closed without merging**